cmd/snap-bootstrap/initramfs_systemd_mount.go

```diff
@@ -75,8 +75,25 @@ type systemdMountOptions struct {
 	Private bool
 	// Umount the mountpoint
 	Umount bool
+	// Overlayfs indicates an overlay filesystem.
+	Overlayfs bool
+	// Directories to be used as lower layers of an overlay mount.
+	// It does not need to be on a writable filesystem.
+	LowerDirs []string
+	// A directory to be used as the upper layer of an overlay mount.
+	// This is normally on a writable filesystem.
+	UpperDir string
+	// A directory to be used as the workdir of an overlay mount.
+	// This needs to be an empty directory on the same filesystem as upperdir.
+	WorkDir string
 }
 
+// forbiddenChars is a list of characters that are not allowed in any mount paths used in systemd-mount.
+const forbiddenChars = `\,:" `
+
+// colons are allowed for the lowerdir option of overlayfs in order to support multiple lowerdir paths.
+const forbiddenCharsForOverlayfsLowerDir = `\," `
```

Comment:
I think this is where we should forbid `:`

Reply:
Pushed a change.